### ISSUE TITLE
One of my pictures isn't showing up

### ISSUE THREAD

**Issue by kingshannon58** [kingshannon58/Baking-Showcase#2]
I am now able to click through the photos using the arrows, but now one of my pictures is not showing up.
[my-baking-showcase (1).zip](https://github.com/kingshannon58/Baking-Showcase/files/3781196/my-baking-showcase.1.zip)

**Comment by kingshannon58** [comment]
I have fixed this issue. I added too many slides and it was making the page assume I had another photo, but I didn't. I also put the numbers of the photos on the same photo and it made it confusing.
![Cake](https://user-images.githubusercontent.com/38970067/67908989-8c829d80-fb4b-11e9-9cf7-c6c26b9c9298.PNG)
![Code](https://user-images.githubusercontent.com/38970067/67908990-8c829d80-fb4b-11e9-91a8-9bcc3495f824.PNG)
